### ISSUE TITLE
Graphical improvement to better cover pattern, concept

### ISSUE THREAD

**Issue by cmoulliard** [snowdrop/snowdrop-site-angular#25]
We should investigate with the graphical designer if we could get such pictures which better explain than 1000 words a concept, pattern, idea to be pat of our guides or home page ... 

<img width="1198" alt="screenshot 2018-07-03 07 00 43" src="https://user-images.githubusercontent.com/463790/42199501-1a2c8316-7e8f-11e8-8400-fbb23c12e0d5.png">

**Comment by lincolnthree** [comment]
Yes, definitely. This is exactly what we have asked Toumas from OSAS for help with.

**Comment by cmoulliard** [comment]
Updates about that ? @lincolnthree

**Comment by lincolnthree** [comment]
Unfortunately not. I have not heard back from Mark N or James Cobb. I'll need to reach out to them again and see if they have time now. I'll do that today.
